package.json

```diff
@@ -4,10 +4,11 @@
 	"description": "",
 	"main": "index.js",
 	"scripts": {
-		"test": "echo \"Error: no test specified\" && exit 1",
+		"test": "mocha",
 		"prepare": "husky install",
 		"start": "node src/index.js",
```

Comment:
i already changed command why it is not update d in your pr
 "build": "tsc",
	"start:server": "npx nodemon src/index.ts"